README.md

```diff
@@ -46,7 +46,7 @@ No additional step is required.
 
 #### Android
 
-1. Open up `android/app/src/main/java/[...]/MainActivity.java`
+1. Open up `android/app/src/main/java/[...]/MainApplication.java`
```

Comment:
I thinks it will be better to explicitly say `application` file, instead if  `MainApplication `. Because users might have this file with other name as well.